src/main/io/ledstrip.h

```diff
@@ -182,6 +183,8 @@ typedef struct ledStripConfig_s {
     uint8_t ledstrip_beacon_armed_only;
     colorId_e ledstrip_visual_beeper_color;
     uint8_t ledstrip_brightness;
+    uint8_t ledstrip_rainbow_delta;
+    uint8_t ledstrip_rainbow_freq;
```

Comment:
Add and initialize in `io/ledstrip.c in `pgResetFn_ledStripConfig``

Reply:
Done, thanks, I didn't know how to set up default values.